sklearn/model_selection/_search_successive_halving.py

```diff
@@ -379,6 +379,9 @@ def _more_tags(self):
                     "Fail during parameter check since min/max resources requires"
                     " more samples"
                 ),
+                "check_estimators_nan_inf": "FIXME",
```

Comment:
These estimators are also tested with others now, and these tests fail. Need fixes in another PR.